xarray/core/parallelcompat.py

```diff
@@ -143,29 +149,72 @@ def get_chunked_array_type(*args) -> ChunkManagerEntrypoint:
 
 class ChunkManagerEntrypoint(ABC, Generic[T_ChunkedArray]):
```

Comment:
I didn't read these closely but you *could* also add cubed links in the "See Also" section. This is very minor though, feel free to skip.